CHANGELOG.md

```diff
@@ -5,6 +5,9 @@ A more detailed list of changes is available in the corresponding milestones for
   - New command-line flag `--skip-network` to skip any checks which require Internet access.
   - Fix number of log level stats colums displayed with --ghmarkdown
 
+### Changes to existing checks
+#### On the Google Fonts Profile
+  - **[com.google.fonts/check/varfont/duplexed_axis_reflow]:** yield one `duplexed_axis_reflow` FAIL per incorrect axis; sort the list of bad glyphs
```

Comment:
changelog entry should be updated to reflect the new check result code-words per axis